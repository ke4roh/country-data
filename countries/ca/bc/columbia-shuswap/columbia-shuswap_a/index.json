{
  "type": "census_subdivision",
  "name": {
    "en": "Columbia-Shuswap A",
    "fr": "Columbia-Shuswap A"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939011",
  "meta": {
    "sgc_province": "59"
  }
}