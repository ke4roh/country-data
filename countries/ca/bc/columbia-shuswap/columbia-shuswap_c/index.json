{
  "type": "census_subdivision",
  "name": {
    "en": "Columbia-Shuswap C",
    "fr": "Columbia-Shuswap C"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939037",
  "meta": {
    "sgc_province": "59"
  }
}