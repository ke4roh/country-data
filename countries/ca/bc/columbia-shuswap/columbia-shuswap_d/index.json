{
  "type": "census_subdivision",
  "name": {
    "en": "Columbia-Shuswap D",
    "fr": "Columbia-Shuswap D"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939039",
  "meta": {
    "sgc_province": "59"
  }
}