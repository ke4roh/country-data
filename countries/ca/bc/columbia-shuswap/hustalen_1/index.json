{
  "type": "census_subdivision",
  "name": {
    "en": "Hustalen 1",
    "fr": "Hustalen 1"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939802",
  "meta": {
    "sgc_province": "59"
  }
}