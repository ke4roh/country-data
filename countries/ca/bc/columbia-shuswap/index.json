{
  "type": "census_subdivision",
  "name": {
    "en": "Columbia-Shuswap",
    "fr": "Columbia-Shuswap"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc",
  "admin_level": 3,
  "local_code": "5939",
  "meta": {
    "sgc_province": "59"
  }
}