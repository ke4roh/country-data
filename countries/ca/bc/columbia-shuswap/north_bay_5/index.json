{
  "type": "census_subdivision",
  "name": {
    "en": "North Bay 5",
    "fr": "North Bay 5"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939803",
  "meta": {
    "sgc_province": "59"
  }
}