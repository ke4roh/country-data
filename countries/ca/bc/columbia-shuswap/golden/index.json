{
  "type": "census_subdivision",
  "name": {
    "en": "Golden",
    "fr": "Golden"
  },
  "alternate_names": [],
  "valid_from": null,
  "valid_to": null,
  "parent_id": "ca/bc/columbia-shuswap",
  "admin_level": 4,
  "local_code": "5939007",
  "meta": {
    "sgc_province": "59"
  }
}